pikaur/config.py

```diff
@@ -105,7 +105,7 @@ def get_config_path() -> str:
         },
         'GnupgHome': {
             'type': 'str',
-            'default': ('/etc/pacman.d/gnupg/' if RUNNING_AS_ROOT else '')
```

Comment:
i actually liked this idea - i meant to document this behavior in the readme

Reply:
> i actually liked this idea - i meant to document this behavior in the readme

I've added it back